po/de.po

```diff
@@ -22823,7 +22823,7 @@ msgstr "weniger Ausgaben"
 
 #: builtin/stash.c:1602 builtin/stash.c:1667
 msgid "include untracked files in stash"
-msgstr "unversionierte Dateien in Stash einbeziehen"
+msgstr "unversionierte Dateien im Stash einbeziehen"
```

Comment:
s.o. Auch ein wenig unglücklich, dass hier für dieselbe Option "in stash" und "in the stash" steht.

Reply:
OK. Es ist nicht ganz dasselbe. Passt schon so.

Reply:
applied 😉 👍